p2-repository/org.eclipse.collections/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+        <name>org.eclipse.collections_8.1.0</name>
+        <comment></comment>
```

Comment:
should the .project file be committed? Or should it be added to .gitignore? 
We already had contributors complain about the Intellij project files.